cmd/saturn-l2/main.go

```diff
@@ -288,6 +294,36 @@ func main() {
 	}
 }
 
+func logL1Connectivity(ctx context.Context, nConnectedL1s *atomic.Uint64) {
+	ticker := time.NewTicker(waitL1Connectivity)
+	defer ticker.Stop()
+
+	isConnected := false
+	isFirstEvent := true
+
+	for {
+		select {
+		case <-ticker.C:
+			if nConnectedL1s.Load() == 0 {
+				if isConnected || isFirstEvent {
+					fmt.Print("ERROR: Saturn Node is not able to connect to the network\n")
+					isConnected = false
+					isFirstEvent = false
+				}
+			} else {
+				if !isConnected || isFirstEvent {
```

Comment:
```suggestion
				if !isConnected {
```

whenever `isFirstEvent` is `true`, `!isConnected` is also true, so it's enough to check for the former